### PR TITLE
mds: change priority of mds rss perf counter to useful

### PR DESCRIPTION
**Pull request by sp98**
mds_cache_mem_rss perf counter can be a useful metric to detect the usage of mds cache and detect mds cache oversize before it happens. The PR changes the priority of the mds_rss metric from Debug to useful

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Comment by vshankar** [comment]
This seems backport worthy. Could you create a tracker and link it in the commit message pls?

File: src/mds/MDSRank.cc [hunk 1]
```diff
@@ -3566,8 +3566,6 @@ void MDSRank::create_logger()
     mdm_plb.add_u64_counter(l_mdm_capa, "cap+", "Capabilities added");
     mdm_plb.add_u64_counter(l_mdm_caps, "cap-", "Capabilities removed");
     mdm_plb.add_u64(l_mdm_heap, "heap", "Heap size");
-
-    mdm_plb.set_prio_default(PerfCountersBuilder::PRIO_DEBUGONLY);
```
Comment (batrick) [inline src/mds/MDSRank.cc:3570]: I would rather not rely on defaults if this is actually wanted by Rook. Please set it to PRIO_USEFUL and add a comment.

**Review by batrick — CHANGES_REQUESTED** [review]

**Review by batrick — APPROVED** [review]

**Review by vshankar — APPROVED** [review]

**Comment by vshankar** [comment]
jenkins test api

**Comment by vshankar** [comment]
I see @avanthakkar testing tag - please approve once tests are done.

**Comment by vshankar** [comment]
@avanthakkar I see your testing tag removed. What does that mean?

**Comment by vshankar** [comment]
> > @avanthakkar I see your testing tag removed. What does that mean?
> 
> My bad! Forgot to mention here that shaman build was successful so removed the testing flag.

ACK. Will put this to test.

**Review by kotreshhr — APPROVED** [review]

**Review by lxbsz — APPROVED** [review]

**Comment by vshankar** [comment]
Merging this without running through integration tests since things are a bit broken there atm.

**Pull request merged**